packages/cli/src/lib/utils/keyValueParser.ts

```diff
@@ -0,0 +1,8 @@
+export default function parseKeyValue(input: string) {
```

Comment:
Seeing that domains had a flag that contained spaced out key/value pairs, I created a utility that could parse that for future flags and or key/value pairs with spaces in them.